### PR TITLE
fix: nettruyen

### PR DESCRIPTION
**Pull request by jckli**
Checklist:
- [x] Updated source's version for individual source changes
- [x] Updated all sources' versions for template changes
- [x] Set appropriate `nsfw` value
- [x] Did not change `id` even if a source's name or language were changed
- [x] Tested the modifications by running it on the simulator or a test device 

just to fix for vus.#6370 lol
cheers
FaWtyz7agAAZxdP

### CODE REVIEW

File: src/rust/wpcomics/sources/nettruyen/res/source.json [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
+    	{
+      		"name": "Hot"
+    	},
+    	{
+      		"name": "Truyện con gái"
+    	},
+   	 	{
+      		"name": "Truyện con trai"
+    	}
+  	]
```
Comment (beerpiss) [inline src/rust/wpcomics/sources/nettruyen/res/source.json:20]: tabs instead of 4 spaces thanks

**Review by beerpiss — CHANGES_REQUESTED** [review]
otherwise just clean up the lint warnings

**Comment by jckli** [comment]
> otherwise just clean up the lint warnings

fuck me why do I have to clean up lint I changed 1 STRING

File: src/rust/wpcomics/sources/nettruyen/res/source.json [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
 		"name": "NetTruyen",
-		"version": 5,
-		"url": "https://www.nettruyenup.com",
+		"version": 6,
+		"url": "https://www.nettruyenvt.com",
 		"nsfw": 1
-	},
-	"listings": [
+  	},
+  	"listings": [
```
Comment (Skittyblock) [inline src/rust/wpcomics/sources/nettruyen/res/source.json:10]: why did you add spaces :uhh:

**Review by Skittyblock — APPROVED** [review]

**Pull request merged**